src/controllers/order.controller.ts

```diff
@@ -51,6 +65,14 @@ class OrderController {
 
     res.json({ orders });
   }
+
+  static getUser(req: Request, res: Response, _next: NextFunction) {
```

Comment:
getUser in each controller? mb you can find better solution without repeating code each time.